awx/ui_next/src/screens/Project/Project.jsx

```diff
@@ -44,6 +44,19 @@ function Project({ i18n, setBreadcrumb }) {
           role_level: 'notification_admin_role',
         }),
       ]);
+
+      if (data.summary_fields.credentials) {
+        const params = {
+          page: 1,
+          page_size: 200,
+          order_by: 'name',
+        };
+        const {
+          data: { results },
+        } = await ProjectsAPI.readCredentials(data.id, params);
+
+        data.summary_fields.credentials = results;
```

Comment:
@AlexSCorey 
Hypothetically speaking, would we need to make this extra api call if `vault_id` were included in the project summary credentials?

Reply:
@jakemcdermott 
no.  if we could add the vault_id to the summary_fields then we wouldn't need to make the extra api calls. In conversations with @wenottingham and @ryanpetrello it seems like there are reasons that adding that info is not possible or not advisable.